homeassistant/components/airnow/sensor.py

```diff
@@ -106,5 +112,22 @@ def extra_state_attributes(self):
             self._attrs[SENSOR_AQI_ATTR_LEVEL] = self.coordinator.data[
                 ATTR_API_AQI_LEVEL
             ]
+            if (
+                SENSOR_AQI_ATTR_FORECAST in self.coordinator.data
+                and self.coordinator.data[SENSOR_AQI_ATTR_FORECAST]
+            ):
+                forecast = self.coordinator.data[SENSOR_AQI_ATTR_FORECAST]
+                self._attrs[SENSOR_AQI_ATTR_FORECAST] = [
+                    {
+                        SENSOR_AQI_ATTR_FORECAST_DATE: fc[
+                            ATTR_API_DATE_FORECAST
+                        ].strip(),
+                        ATTR_API_AQI: fc[ATTR_API_AQI],
+                        SENSOR_AQI_ATTR_FORECAST_LEVEL: fc[ATTR_API_CATEGORY][
+                            ATTR_API_CAT_LEVEL
+                        ],
+                    }
+                    for fc in forecast
+                ]
```

Comment:
These do not belong in a sensor, instead, use a `weather` entity.

Reply:
I'd love to use a `weather` entity, but it looks like `condition` and `temperature` is currently required on those (per https://developers.home-assistant.io/docs/core/entity/weather). As far as I understand, it'd require a schema change to that entity type to allow air-quality-only weather entities. Should we go down that path?

Reply:
That requires an architectural discussion.

In the mean time, I don't think we should be adding all this information to attributes in sensors. In general, we are trying to reduce the use of attribute (not add to them).

Reply:
Opened https://github.com/home-assistant/architecture/discussions/879 to discuss the entity model change. Thanks!